### PR TITLE
Remove dead-code and address snake_case violations and apply Clang format when possible

### PR DESCRIPTION
**Pull request by hassount**
## Description:

- Remove dead code
- Address some snake_case PascalCase violations for variables and types
- Rename lcu fields to sb
- Rename tu fields to txb
- Rename cu fields to blk
- Shorten variable names in prep to apply clang format: e.g. 
picture_control_set_ptr -- > pcs_ptr
sequence_control_set_ptr -- > scs_ptr
- Remove RawStringFormats property from clang-format until it's fixed
- Apply clang-format on files in source/lib/encoder/codec/ except:
aom_dsp_rtcd.c
aom_dsp_rtcd.h
EbMdRateEstimation.h
EbIntraPrediction.c
EbModeDecisionConfiguration.c
EbMotionEstimation.h
EbPictureDecisionProcess.c
EbRateControlTables.c
EbQMatrices.h
EbEncHandle.c

## Performance:

No Impact

### CODE REVIEW

File: Source/Lib/Common/Codec/EbDefinitions.h [hunk 1]
```diff
@@ -87,48 +85,50 @@ typedef struct {
...
 /****************** Pre-defined Values ******************/
 /********************************************************/
 
-#define ALTREF_MAX_NFRAMES 10 // maximum number of frames allowed for the Alt-ref picture computation
-                              // this number can be increased by increasing the constant
-                              // FUTURE_WINDOW_WIDTH defined in EbPictureDecisionProcess.c
+#define ALTREF_MAX_NFRAMES                                                     \
+    10 // maximum number of frames allowed for the Alt-ref picture computation \
+        // this number can be increased by increasing the constant             \
+        // FUTURE_WINDOW_WIDTH defined in EbPictureDecisionProcess.c
```
Comment (1480c1) [inline Source/Lib/Common/Codec/EbDefinitions.h]: Perhaps this comment can be separated into it's own line and using c style block comments?

Comment (hassount) [reply Source/Lib/Common/Codec/EbDefinitions.h]: good catch, @JackZhouVCD while you're at it

**Comment by 1480c1** [comment]
Also, I would recommend to add

```diff
diff --git a/.github/workflows/style-check.yml b/.github/workflows/style-check.yml
index 858ffbcc..193ec228 100644
--- a/.github/workflows/style-check.yml
+++ b/.github/workflows/style-check.yml
@@ -21,7 +21,7 @@ jobs:
         run: git fetch -q https://github.com/OpenVisualCloud/SVT-AV1.git master
       - name: newline at EOF check
         run: |
-          for file in $(git diff --name-only FETCH_HEAD  -- . ':!third_party' ':!test/e2e_test/test_vector_list.txt' ':!test/vectors/smoking_test.cfg' ':!test/vectors/video_src.cfg' ':!*.png'); do
+          for file in $(git diff --name-only --diff-filter=d FETCH_HEAD  -- . ':!third_party' ':!test/e2e_test/test_vector_list.txt' ':!test/vectors/smoking_test.cfg' ':!test/vectors/video_src.cfg' ':!*.png'); do
             if [ -n "$(tail -c1 "$file")" ]; then
               echo "No newline at end of $file"
               false
```

to prevent style-check from erroring out on deleted files

File: Source/Lib/Common/Codec/EbDefinitions.h (hunk 1 above)
Comment (JackZhouVCD) [reply Source/Lib/Common/Codec/EbDefinitions.h]: @1480c1 @hassount yup, I'll include this in the coming commit

**Pull request merged**